extras/nightly-container/provision.yml

```diff
@@ -77,11 +77,37 @@
       args:
         warn: no
 
+    - name: Create override directory for glusterd2.service
+      file:
+        path: /etc/systemd/system/glusterd2.service.d
+        state: directory
+
+    # This makes systemd pass environment variables set for GD2 by kubernetes
```

Comment:
@kshlm, I think this is not only meant for kubernetes.  can we update the comments saying any CO can set this environment variables,